doc/source/cluster/kubernetes/user-guides/configuring-autoscaling.md

```diff
@@ -313,6 +313,50 @@ These fields specify Autoscaler container environment variables.
 These fields should be formatted following the [Kubernetes API](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/Pod-v1/#environment-variables)
 for container environment variables.
 
+### 4. Set the `rayStartParams` and the resource limits for the Ray container
+
+The Ray Autoscaler reads the `rayStartParams` field or the Ray container's resource limits in the RayCluster custom resource specification to determine the Ray Pod's resource requirements.
```

Comment:
does Ray Autoscaler actually care about `Ray container's resource limits in the RayCluster custom resource specification`?

Reply:
Yes

Reply:
You can see the example for more details. If you comment out the CPU resource limit of the worker group, the Ray Autoscaler would fail.

Reply:
oh - i see, i feel the example is a bit confusing to me: nothing was commented out. 

So it's like:
- If users set `rayStartParams`: things would work (regardless of resources limits on the containers)
- if users don't set `rayStartParams`: one has to set the resource limits on the containers? 

If so, i feel calling out these two explicitly might help.

Reply:
Good idea. I will update the PR.

Reply:
Updated